app/src/main/res/values/strings.xml

```diff
@@ -81,4 +81,7 @@
     <string name="github" translatable="false">GitHub</string>
     <string name="github_summary">Check out our GitHub repository and help us improve the app</string>
     <string name="github_repository_url" translatable="false">https://github.com/bk20dev/forest</string>
+    <string name="version">Version: v1.0.0</string>
```

Comment:
It seems like it's possible to get the version name from the build.gradle file. This way, there would be no chance of forgetting to update version codes.
https://stackoverflow.com/questions/4471025/how-can-you-get-the-manifest-version-number-from-the-apps-layout-xml-variable
This approach will probably require writing some kind of dynamic Preference generation/altering rendered preferences. 
Also could you make the first line say {AppName} v{VersionName}, like Forest v1.0.

Reply:
I would love to set this up, but sadly I am not able to understand what I should be doing. I am new to android development, sorry for the inconvenience

Reply:
So, Forest uses Gradle to automate its build process (it's the default option). Under Gradle Scripts tab in Android Studio you will find a build.gradle (Module :app) file. It contains information about the app dependencies as well as some "meta data", including the current application version.
The StackOverflow answer I linked shows how to take that information and make it accessible as a string resource, which you can then show to the user somewhere. For current version of Forest, this string resource should resolve to "1.0" as set in build.gradle.
I would love to see the app version as a tile with two lines. The second line you provided looks okay, but make the first line ("header") display "Forest v1.0". Make it flexible, so the app name (Forest) and version name (1.0) are not hardcoded, but rather taken from strings.xml and build.gradle (as a string resource). I'm not sure if it is possible to interpolate two strings in strings.xml, but it's surely possible to do from code. You will need to find a place (SettingsFragment.kt) to write some string-interpolation logic and set it in the preference fragment. Check out the code i wrote there, because it shows how to access a preference.